.pre-commit-config.yaml

```diff
@@ -12,21 +12,12 @@ repos:
       - id: end-of-file-fixer
       - id: mixed-line-ending
       - id: trailing-whitespace
-  - repo: https://github.com/timothycrosley/isort
-    rev: 5.12.0
-    hooks:
-      - id: isort
```

Comment:
why was `isort` removed? is sorting covered by new ruff version?

Reply:
yup! it does it all, it's pretty amazing.

Reply:
replaces pylint, isort, yapf and autoflake which we used :)